vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -327,7 +327,14 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
             layer.w13_weight.data = shuffled_w13
             layer.w2_weight.data = shuffled_w2
 
-        if current_platform.is_cpu():
+        if current_platform.is_xpu():
+            import intel_extension_for_pytorch as ipex
+            layer.ipex_fusion = ipex.llm.modules.GatedMLPMOE(
```

Comment:
can we follow code path like cpu, add a `xpu_fused_moe.py`. I think we will also add another path in the future via vllm-xpu-ops.

Reply:
But we probably won't keep both paths. I am not sure what else can be put in `xpu_fused_moe.py` and for now, we may keep like this for this single method?

Reply:
for test/benchmark purpose, we may keep both. it's ok to keep it now.